src/components/common-components/drawer/drawer.component.tsx

```diff
@@ -1,22 +1,71 @@
-import { Drawer as MaterialDrawer, DrawerProps, IconButton } from "@material-ui/core";
+import {
+  Box,
+  Divider,
+  Drawer as MaterialDrawer,
+  DrawerProps,
+  Grid,
+  IconButton,
+} from "@material-ui/core";
 import { MdClose } from "react-icons/md";
-
 import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import ScssVars from "../../../assets/styles/styles/custom/_variables.module.scss";
+
+const useStyles = makeStyles({
+  drawer: {
+    minWidth: 500,
+  },
+  drawerContentMargin: {
+    paddingTop: 25,
+    paddingLeft: 24,
+    paddingRight: 24,
+    paddingBottom: 15,
+  },
+  exitButton: {
+    margin: "-7px -8px",
+    marginTop: -15,
+    color: ScssVars.primary,
+  },
+  title: {
+    fontFamily: ScssVars.fontFamilyPrimary,
+    fontWeight: 700,
+    fontSize: 18,
```

Comment:
Ten size też możesz wziąc ze stałych. To chyba font size z końcówką lg, ale nie jestem pewien na 100%